dashboard/modules/event/event_consts.py

```diff
@@ -16,8 +16,4 @@
 EVENT_AGENT_RETRY_TIMES = 10
 EVENT_AGENT_CACHE_SIZE = 10240
 # Event sources
-EVENT_HEAD_MONITOR_SOURCE_TYPES = [event_pb2.Event.SourceType.Name(event_pb2.Event.GCS)]
```

Comment:
I removed it cuz I thought it is unnecessary. It is used to decide what event files head / worker node agent can monitor. Removing it means we will monitor all log files within `logs/event`